app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -111,6 +164,45 @@ export const ListingDetailsEligibility = ({
         <StandardTable headers={occupancyTableHeaders} data={occupancyTableData} />
       </ListSection>
       <ListingDetailsPreferences listingID={listing.listingID} />
+      {priorityUnits?.length > 0 ? (
+        <ListSection
+          title={t("listings.priorityUnits")}
+          subtitle={t("listings.priorityUnitsDescription")}
+        >
+          {priorityUnits
+            .filter((unit: ReducedUnit) => {
+              return unit?.name !== "Adaptable"
+            })
+            .map((unit: ReducedUnit) => {
```

Comment:
I noticed a slightly different ordering (the last two are swapped). Any idea what's going on here?
![image](https://user-images.githubusercontent.com/98352600/187276318-9acc09cf-25a0-49f0-a8ed-b98128720261.png)

Reply:
just an fyi - I updated the ticket's Review Instructions re: this. We looked into why this is the case and nothing was obvious and didn't want to spend too much time on something that might not be be desired. The ordering was built in a way that mirrored Salesforce.